### PR TITLE
removing unwanted views and checking for instant game

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: src/device.js [hunk 1]
```diff
@@ -61,10 +56,8 @@ if (xdpi && ydpi) {
 }
 
 exports.get = function (module) {
-  // deprecated: InputPrompt used to be platform-specific
-  if (module == 'InputPrompt') { return jsio('import ui.InputPrompt'); }
+  var path = 'platforms.browser';
 
-  var path = _devices[exports.name] || 'platforms.browser';
   return jsio('import ' + path + '.' + module, {dontExport: true, suppressErrors: true});
```
Comment (Javasamurai) [inline src/device.js]:
```suggestion
   return jsio('import platforms.browser.' + module, {dontExport: true, suppressErrors: true});
```

File: src/device.js [hunk 2]
```diff
@@ -61,10 +56,8 @@ if (xdpi && ydpi) {
 }
 
 exports.get = function (module) {
-  // deprecated: InputPrompt used to be platform-specific
-  if (module == 'InputPrompt') { return jsio('import ui.InputPrompt'); }
+  var path = 'platforms.browser';
```
Comment (Javasamurai) [inline src/device.js]: We don't even need this

Comment (rampr) [reply src/device.js]: Why is this not needed Syed?

Comment (Javasamurai) [reply src/device.js]: I mean the variable itself.

Comment (Javasamurai) [reply src/device.js]: Check the above comment.

**Review by Javasamurai — APPROVED** [review]

**Review by rampr — APPROVED** [review]

**Pull request merged**